### ISSUE TITLE
Fix possible ender pearl issue with being ale to insta teleport

### ISSUE THREAD

**Issue by rourke750** [Civcraft/Humbug#27]
http://www.reddit.com/r/Civcraft/comments/34k2nz/will_enderpearl_teleportion_machineteleporter_be/cqvl6ud

**Comment by ttk2** [comment]
yup, thanks for making the issue, should we just check pearl events for the
distance thrown? I wonder if there is any way to limit our check overhead
to just this one situation.

**Comment by rourke750** [comment]
Every entity has an id.  I'm going to schedule a timer and if it shows up
I'm just going to delete it.

**Comment by rourke750** [comment]
possibly fixed https://github.com/Civcraft/Humbug/commit/031c832406ea45ce1bbd52c116b571c587d9eae9

**Comment by ttk2** [comment]
this should be resolved right?

**Comment by Xavter** [comment]
Me and sanwi will build a pearl machine later, we can double check it then

**Comment by suirad** [comment]
Was this tested?

**Comment by suirad** [comment]
@Xavter did you happen to test this?
